src/igvfd/schemas/whole_organism.json

```diff
@@ -68,7 +68,13 @@
     },
     "properties": {
         "schema_version": {
-            "default": "10"
+            "default": "11"
+        },
+        "part_of": {
+            "type": "null"
```

Comment:
Could you leave a comment explaining that this property should not be submitted.

Reply:
these properties*

Reply:
Good idea, added